Library/Formula/pngcheck.rb

```diff
@@ -3,10 +3,11 @@
 class Pngcheck < Formula
   homepage 'http://www.libpng.org/pub/png/apps/pngcheck.html'
   url 'https://downloads.sourceforge.net/project/png-mng/pngcheck/2.3.0/pngcheck-2.3.0.tar.gz'
+  revision 1
   sha1 'e7f1535abbf2f809e036a9a43c759eeac5e39350'
 
   def install
-    system 'make pngcheck'
-    bin.install 'pngcheck'
+    system 'make -f Makefile.unx ZINC=-I/usr/include ZLIB=/usr/lib/libz.dylib'
+    bin.install %w(pngcheck pngsplit png-fix-IDAT-windowsize)
```

Comment:
` %w[pngcheck pngsplit png-fix-IDAT-windowsize] ` is preferred, or ` %w{ pngcheck pngsplit png-fix-IDAT-windowsize } `.